test/externalTest.js

```diff
@@ -38,7 +38,7 @@ mineflayer.supportedVersions.forEach((supportedVersion, i) => {
   const PORT = Math.round(30000 + Math.random() * 20000)
   const mcData = require('minecraft-data')(supportedVersion)
   const version = mcData.version
-  const MC_SERVER_JAR_DIR = process.env.MC_SERVER_JAR_DIR
+  const MC_SERVER_JAR_DIR = process.env.MC_SERVER_JAR_DIR || `${process.cwd()}/server_jars`
```

Comment:
Why ?

Reply:
I think we should have some default option

Reply:
Ah, in case you run the tests locally, ok makes sense